libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.EditActions.schema

```diff
@@ -1,8 +1,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "EditActions Action",
-    "description": "Edit current dialog with changeType and Actions",
+    "title": "Change conversation flow",
```

Comment:
>Change conversation flow [](start = 14, length = 24)

What? Why?

Reply:
This is what it does: this action changes the conversation flow.
While it possibly makes a lot of sense to those that are deeply familiar with the SDK, that is not the audience that will most need/benefit from the description (they already know!) - those that are still learning the SDK are.